randovania/gui/ui_files/games_tab_dread_widget.ui

```diff
@@ -300,14 +300,14 @@ Save in the Save Station immediately west of Red Teleportal.
          <x>0</x>
          <y>0</y>
          <width>631</width>
-         <height>465</height>
+         <height>468</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="hints_scroll_layout">
         <item>
          <widget class="QLabel" name="hints_label">
           <property name="text">
-           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;In Metroid Dread, you can find hints from the following sources:&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Network Station&lt;/span&gt;: Contains one unique item hint per station. A single item can't be hinted twice by two network stations.&lt;/p&gt;&lt;hr/&gt;&lt;p align=&quot;justify&quot;&gt;For more details with how hints are assigned, check Metroid Prime 2: Echoes page. Hints assigned to Network Stations follow the same logic as item hints assigned to Luminoth Lore.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;In Metroid Dread, you can find hints from the following sources:&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Network Station&lt;/span&gt;: Contains one unique item hint per station. A single item can't be hinted twice by two network stations.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dairon - Navigation Station North&lt;/span&gt;: ADAM will provide a hint to the location of all Metroid DNA.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Itorash&lt;/span&gt;: A new Navigation Station has been added to the elevator room. ADAM will provide a hint to the location of all Metroid DNA.&lt;/p&gt;&lt;hr/&gt;&lt;p align=&quot;justify&quot;&gt;For more details with how hints are assigned, check Metroid Prime 2: Echoes page. Hints assigned to Network Stations follow the same logic as item hints assigned to Luminoth Lore.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Shouldn't this be `hint to the locations of all Metroid DNA`. So, the plural of `location` because there are multiple locations? (Occurs two times)